doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -195,11 +204,8 @@ Requests to the environment
    This command displays the type of :n:`@term`. When called in proof mode, the
    term is checked in the local context of the current subgoal.
 
-   .. cmdv:: @selector: Check @term
-
-      This variant specifies on which subgoal to perform typing
-      (see Section :ref:`invocation-of-tactics`).
-
+   Prefix the command with :n:`@selector:` to specify which subgoal to check.
```

Comment:
```suggestion
   Prefix the command with :n:`@selector:` to specify which subgoal to check
```

Like I said in one of my previous comments, it'd be great if the section could be consolidated to include all (and only those) commands that accept the goal selector, so that this can be documented once and for all at the top of the section.